module/VuFind/src/VuFind/Db/Service/CommentsService.php

```diff
@@ -165,4 +165,45 @@ public function changeResourceId(int $old, int $new): void
     {
         $this->getDbTable('comments')->update(['resource_id' => $new], ['resource_id' => $old]);
     }
+
+    /**
+     * Get a paginated result of all comments by user id
+     *
+     * @param int    $userId User Id
+     * @param int    $limit  Limit
+     * @param int    $page   Page
+     * @param string $sort   Sort
+     *
+     * @return Paginator
+     */
+    public function getCommentsPaginator(
+        int $userId,
+        int $limit,
+        int $page,
+        string $sort,
+    ): \Laminas\Paginator\Paginator {
+        return $this->getDbTable('Comments')->getCommentsPaginator(
+            $userId,
+            $limit,
+            $page,
+            $sort,
+        );
+    }
+
+    /**
+     * Delete comments by given user and comment ids
```

Comment:
Add punctuation for consistency.

```suggestion
     * Delete comments by given user and comment ids.
```